projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -191,6 +191,11 @@ public static function load_wpcom_user_features() {
 		require_once __DIR__ . '/features/wpcom-sidebar-notice/wpcom-sidebar-notice.php';
 		require_once __DIR__ . '/features/wpcom-themes/wpcom-themes.php';
 
+		// We include WPCom Themes results and installation on WoA sites only and non-WP_CLI context.
+		if ( ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) && ( ! defined( 'WP_CLI' ) || ! WP_CLI ) ) {
```

Comment:
Would it be an option to rely on `( new \Automattic\Jetpack\Status\Host() )->is_woa_site()` here, just like for the check just below?

Reply:
I'm not sure! This condition is trying to replicate the exact logic we had before when the files were in wpcomsh. As long as `( new \Automattic\Jetpack\Status\Host() )->is_woa_site()` works the same way I'm sure it's fine. What's the benefit?

Reply:
The main benefit would be that you would only have to call that method once, and then use it in both places It would be cleaner and a bit better performance-wise.

The method is a bit different though; it checks for different constants (`ATOMIC_SITE_ID` and `ATOMIC_CLIENT_ID` instead of `IS_ATOMIC`, and checks that the wpcomsh plugin is active with the `WPCOMSH__PLUGIN_FILE` constant. Overall it's a bit more precise of a check.

Reply:
When we tried using `( new \Automattic\Jetpack\Status\Host() )->is_woa_site()` the test failed, so we reverted to the `IS_ATOMIC` check